src/main/java/org/opentripplanner/apis/gtfs/datafetchers/ItineraryImpl.java

```diff
@@ -71,6 +71,11 @@ public DataFetcher<Iterable<Leg>> legs() {
     return environment -> getSource(environment).getLegs();
   }
 
+  @Override
+  public DataFetcher<Integer> numberOfTransfers() {
+    return environment -> getSource(environment).getNumberOfTransfers();
```

Comment:
Do we know if this is always non null in the itinerary or do we have to add some 0 default value here?

Reply:
The calculation happens here: https://github.com/opentripplanner/OpenTripPlanner/blob/027556c62551fb677591195562bd68edaf029718/src/main/java/org/opentripplanner/model/plan/ItinerariesCalculateLegTotals.java#L29-L67

Reply:
Seems like it uses 0 as default which is ok.

Reply:
Then the count will be 0.

There is also a test for this: https://github.com/opentripplanner/OpenTripPlanner/blob/1e397d85265d64efcef71e8c524f03de6d1bc6a6/src/test/java/org/opentripplanner/model/plan/ItineraryTest.java#L21-L43